scripts/patch-emsdk.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+PATCH_FILE="./emscripten.patch"
+EMSCRIPTEN_DIR="$(dirname $(which emcc))"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)